app/code/Magento/CustomerGraphQl/Model/Customer/ValidateCustomerData.php

```diff
@@ -15,6 +18,11 @@
  */
 class ValidateCustomerData
 {
+    /**
+     * @var ValidateCustomerDataInterface[]
+     */
+    protected $validators = [];
```

Comment:
Please change the variable to private.

Reply:
Thank you for the explanations, @dthampy !

**done**